src/app/(dashboard)/layout.tsx

```diff
@@ -49,6 +50,9 @@ const Layout = async ({children}: LayoutProps = {children:null})=>{
                     <li>
                         <FriendRequestSidebarOptions initialRequestCount={friendRequests.length} sessionId={session.user.id}/>
                     </li>
+                    <li>
+                        <SignOutButton/>
+                    </li>
```

Comment:
Like how clean this is... however, can this either be included in the FriendRequestSidebar Options or simplify that iteratble options to a component? An array of one is futureproofing against jack and squat.